pkgs/development/libraries/nss/default.nix

```diff
@@ -52,7 +56,15 @@ in stdenv.mkDerivation rec {
     "NSS_ENABLE_ECC=1"
     "USE_SYSTEM_ZLIB=1"
     "NSS_USE_SYSTEM_SQLITE=1"
-  ] ++ stdenv.lib.optional stdenv.is64bit "USE_64=1"
+  ] ++ stdenv.lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) (let
+    cpu = stdenv.hostPlatform.parsed.cpu.name;
+  in [
+    "NATIVE_CC=${buildPackages.stdenv.cc}/bin/cc"
+    "CROSS_COMPILE=1"
+    "NSS_DISABLE_GTESTS=1" # don't want to build tests when cross-compiling
+    "OS_TEST=${cpu}"
+    "CPU_ARCH=${cpu}"
```

Comment:
We should be able to use NATIVE_CC, OS_TEST and CPU_ARCH unconditionally, no?
This way we have less code that is only when cross-compiling.

Reply:
updated!

Reply:
Wait was this done?